pages/index.js

```diff
@@ -46,7 +47,7 @@ function Home({ states }) {
                 <div className={`col-12 mb-4${mapClass}`}>
                     <div className='float-right'>
                         <div><small>Survivor wealth friendliness</small></div>
-                        <div id='legend' style={{height: '40px', width: '200px', backgroundColor: 'blue'}}/>
+                        <Image src="/images/legend.png" height="40" width="200"/>
```

Comment:
I think this looks great but I'm not sure how accessible it is. I'm wondering whether users with screen readers will need a legend (e.g. will the ranking of each state just be read out for them based on the map data?). Also wondering if this should have an `alt` tag. @DanielKim1 probably knows more than me!

Reply:
Good catch Emily! Looks like NextJS does _not_ automatically handle accessibility for us.
I'm thinking that the legend is useless for screen reader users anyway:
<img width="250" alt="Screen Shot 2021-03-10 at 9 17 11 AM" src="https://user-images.githubusercontent.com/22206686/110643656-5400c400-8182-11eb-97a4-6585492c0cc0.png">
So I think we should just hide the whole thing from screen readers by adding the `aria-hidden="true"` attribute to `div.float-right`.

Reply:
Thanks! @DanielKim1 could you take a quick look at my latest commit to make sure I added the attribute correctly?

Reply:
looks perfect!